src/domain/data-decoder/entities/data-decoded.entity.ts

```diff
@@ -8,4 +8,5 @@ export interface DataDecodedParameter {
 export interface DataDecoded {
   method: string;
   parameters: DataDecodedParameter[] | null;
+  humanDescription?: string;
```

Comment:
For this project, we have decided to use `null` to be more explicit on the absence of value.

`undefined` was considered but ultimately for domain classes we wanted to be more explicit that the property has no value (and not that it was not "set").